tests/appsec/test_alpha.py

```diff
@@ -34,7 +34,6 @@ def test_uri(self):
         r = self.weblog_get("/waf/0x5c0x2e0x2e0x2f")
         interfaces.library.assert_waf_attack(r, pattern="0x5c0x2e0x2e0x2f", address="server.request.uri.raw")
 
-    @bug(library="python@1.1.0", reason="a PR was not included in the release")
     def test_headers(self):
```

Comment:
If I'm not wrong, the bug is shipped in this exact version, so we need to keep this decorator.

Reply:
:+1: ok, I  I thought bug decorator affects to "greater or equal" versions, no "lower or equal", Thank you!